docs/platforms/python/performance/instrumentation/custom-instrumentation/queues-module.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: Instrument Queues
+sidebar_order: 3000
+description: "Learn how to manually instrument your code to use Sentry's Queues module. "
+---
+To ensure that you have performance data about your messaging queues, you'll need to instrument custom spans and transactions around your queue producers and consumers.
```

Comment:
I'd mention somewhere that this only needs to be done if whatever queuing library you're using isn't instrumented by Sentry out of the box. Maybe something like:

```suggestion
Sentry comes with automatic instrumentation for the most common messaging queue systems. In case yours isn't supported, you can still instrument custom spans and transactions around your queue producers and consumers to ensure that you have performance data about your messaging queues.
```